Fred2/IO/EnsemblAdapter.py

```diff
@@ -214,7 +214,7 @@ def write_seqs(self, name):
             :param str name: The complete path with file name where the fasta is going to be written
             """
         with open(name, "w") as output:
-            SeqIO.write(self.collection.values(), output, "fasta")
+            SeqIO.write(list(self.collection.values()), output, "fasta")
```

Comment:
list probably not needed?

Reply:
Since it needs a list or an iterator I would leave it

Reply:
removed the list